msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/states/AccountState.kt

```diff
@@ -220,18 +221,67 @@ class AccountState private constructor(
     }
 
     /**
-     * Retrieves the access token for the currently signed in account from the cache.
+     * Retrieves the access token for the default OIDC scopes from the cache.
```

Comment:
Developers won't know what this means. Let's make it more explanatory and developer friendly, please. Applies also to other instances of this comment.

Reply:
Done